### DISCUSSION TITLE
Data for update on Docker Hub about "Fix unwanted location in badge SVG generation"

### DISCUSSION THREAD

**Discussion by gui-angelo** [healthchecks/healthchecks#651] [General]
Hello good morning, 

I would like to know when the image on the Healthchecks Docker Hub regarding [Fix unwanted localization in badge SVG generation](https://github.com/healthchecks/healthchecks/commit/c4bbb4432a9dd24bbc4d22a23a16d9cd53cab028) will be updated. Would you have any predictions of this code change being updated in the Docker Hub image?

**Comment by cuu508** [comment]
Hello, a new Docker image is published on every release. There is no strict release schedule, I make releases whenever I feel there's a good amount of new stuff to release. I plan to make the next release (v2.1) sometime next week.

**Reply by gui-angelo** [reply]
Thanks a lot for the feedback. I'm waiting for the update. 
Have a good week!

**Reply by cuu508** [reply]
v2.1 is out, and new images are available on Docker Hub

**Reply by gui-angelo** [reply]
Great news, thank you very much for your attention. I will be deploying in my environment.
